identity/aziot-identityd/src/lib.rs

```diff
@@ -28,6 +28,27 @@ const DPS_ASSIGNMENT_RETRY_INTERVAL_SECS: u64 = 10;
 /// This is the number of seconds to wait for DPS to complete assignment to a hub
 const DPS_ASSIGNMENT_TIMEOUT_SECS: u64 = 120;
 
+/// URI query parameter that identifies module identity type.
+const ID_TYPE_AZIOT: &str = "aziot";
+
+/// URI query parameter that identifies local identity type.
+const ID_TYPE_LOCAL: &str = "local";
+
+macro_rules! match_id_type {
```

Comment:
```rust
($id_type:ident { $( $type:ident => $action:expr ,)+ })
```

will allow you to write:

```rust
match_id_type!(id_type {
    ID_TYPE_AZIOT => self.id_manager.get_module_identity(module_id).await,
    ID_TYPE_LOCAL => self.issue_local_identity(module_id).await,
});
```

and so on. That looks closer to the regular `match` syntax, and allows trailing commas on all arms.

Reply:
Fixed, thanks!